site/contents/write_ups/2022/12_01_22-writeup.md

```diff
@@ -0,0 +1,93 @@
+
+Vanessa hopes to run some practical sessions during Data Ethics meetings. Vanessa plans on writing a hate piece on the Metaverse, a piece we are all very excited for. Vanessa is also going to look into [Is debiasing bad?](https://edri.org/our-work/if-ai-is-the-problem-is-debiasing-the-solution/), understand the arguments and potentially make use of them in her thesis if relevant. Roman wants to write an article about ethics education in computer science.
+
+Nina wants to publish a [Data Hazards](https://very-good-science.github.io/data-hazards/) analysis of her projects on the Open Science Framework.
+
+## Balancing saltiness and sweetness
+
+Ola's resolution is to, as she put it, balance saltiness and sweetness in her approach to academic reading. This is not just because of politeness. Rather than being bitter and defensive against academic reading, is there something to do be done to make the reading about the author and not your own ego? How do we learn to love reading again and not see it as a chore? 
+
+Ola will be trying to take a slightly more positive outlook on things whilst still being thorough. Paper reviewers could benefit from this too! This idea resonated with lots of members of the group, who missed the days when they could genuinely read for joy instead of just for work. Unfortunately PhD students are just being optimised for picking out "what can I get out of this?" from any source they read from
+
+Ola also mentioned the [civic labratory](https://civiclaboratory.nl/2021/01/03/collabrary-a-methodological-experiment-for-reading-with-reciprocity/), a group asking themselves: "How can we change academic reading relations that tend to be extractive into something more reciprocal, humble, generous, and accountable?"
+
+This led us to asking if there is a dichotomy between being nice and being pessimistic? You don't always have to read to evaluate, unless you're doing a systematic literature review. You can just read, as if it was a fiction book. Maybe asking different questions might make it better. Can you read for how the author arrived at arguments? What claims did they make? What were they supported by?
+
+This is not just about being a nicer person. If your own wellbeing suffers from being a bit salty, maybe you need to learn to enjoy reading again. Film snobs tend to enjoy films less, maybe we reading snobs should learn to enjoy reading again.
+
+## Blowing the Whistle and Looking at Jobs
+
+One of our members, Euan has intents to "blow the whistle" this year in his field. His studies are not being listened to, even though it would make things safer in his field. Euan intends to try and continue to look for opportunities to apply more of what we learn at Data Ethics Club to what he is doing.
```

Comment:
```suggestion
One of our members, Euan, has intents to "blow the whistle" this year in his field. 
His studies are not being listened to, even though it would make things safer in his field. 
Euan intends to try and continue to look for opportunities to apply more of what we learn at Data Ethics Club to what he is doing.
```